src/fprime_gds/common/fpy/grammar.lark

```diff
@@ -76,13 +99,25 @@ NAME: /[^\W\d]\w*/
 COMMENT: /#[^\n]*/
 CONST_TRUE: "True"
 CONST_FALSE: "False"
+ADD_OP: "+"
+SUB_OP: "-"
+DIV_OP: "/"
+MUL_OP: "*"
+FLOOR_DIV_OP: "//"
+MOD_OP: "%"
+POW_OP: "**"
+OR_OP: "or"
+AND_OP: "and"
+NOT_OP: "not"
 COMPARISON_OP: ">" | "<" | "<=" | ">=" | "==" | "!="
 
 STRING: /("(?!"").*?(?<!\\)(\\\\)*?"|'(?!'').*?(?<!\\)(\\\\)*?')/i
 
-_SPECIAL_DEC: "0".."9"        ("_"?  "0".."9"                       )*
 DEC_NUMBER:   "-"? "1".."9"        ("_"?  "0".."9"                       )*
           |   "-"? "0"             ("_"?  "0"                            )* /(?![1-9])/
 
+_SPECIAL_DEC: "0".."9"        ("_"?  "0".."9"                       )*
```

Comment:
Can we remove `-` from the syntax of literals and use unary minus instead? Or is that future work?

Reply:
OK, but unary minus is allowed, right? So if a user writes `- 1` it will generate those instructions. It seems brittle to make an optimization like this depend on spacing. I think it would be better to be consistent, and just accept the fact that the code is unoptimized for now.

Reply:
Sounds good. I think the docs need to be updated also.